hooks/blueprint

```diff
@@ -36,7 +39,7 @@ if new_enough "$cf_kit_version" 2.3.0 ; then
     "overlay/enable-log-cache.yml"
     "overlay/instance-identity-cert-from-cf.yml"
     "overlay/loggregator-certs-from-cf.yml"
-    "overlay/log-cache-syslog-server.yml"
+#    "overlay/log-cache-syslog-server.yml" deprecated  - default from upstream
```

Comment:
Just delete this line if its no longer used

Reply:
done